src/styles/WordTestList.scss

```
@@ -0,0 +1,70 @@
+@import './variables/colors.scss';
+@import './variables/textStyles.scss';
+
+.babo {
+    display: flex;
+    justify-content: center;
+    border-bottom: 1px solid;
+    padding-bottom: 16px;
+    margin-bottom: 12px;
+    width: calc(100vw - 40px);
+    margin-left: auto;
+    margin-right: auto;
+    color: $koreasy-color-gray-03;
+}
+
+.test-button {
+    @include body01-bold;
+    background-color: $koreasy-color-primary;
+    color: $koreasy-color-white;
+    border: none;
+    margin-right: 12px;
+    margin-left: 20px;
+    width: calc((100vw - 52px) / 2);
+    height: 40px;
+    border-radius: 12px;
+    font-size: medium;
+}
+
+.learn-button {
+    @include body01-bold;
+    border: none;
+    background-color: $koreasy-color-gray-06;
+    color: $koreasy-color-primary;
+    width: calc((100vw - 52px) / 2);
+    margin-right: 20px;
+    height: 40px;
+    border-radius: 12px;
+    font-size: medium;
+}
+
+.word-test-list-view-name {
+    @include title01-heavy;
+    color: $koreasy-color-primary;
+    margin-left: 20px;
+    // margin-bottom: 12px;
+}
+
+.word-test-main-container {
+    display: flex;
+    flex-direction: column;
+}
+
+.word-test-list-title {
+    padding-left: 0px;
+    align-items: center;
+}
+
+.big-title {
+    @include title02-bold;
+    color: $koreasy-color-black;
+    margin-left: 20px;
```

```suggestion
    margin-bottom: 4px;
```
`p` 태그를 사용하면 margin-bottom에 `1rem`이라는 공백이 생기게 됩니다.
이 경우를 방지하기 위해 margin 값을 초기화해주셔야합니다.
아래 리뷰에 후술하겠습니다.

---

제가 확인하였을 때, big-title과 small-title에서 `p` 태그가 아닌 `div`태그를 사용한 것으로 보입니다.
혹시 제가 놓치고 있는 부분이 있을 수도 있을 것 같은데, `scss` 파일이 아닌 `jsx` 파일에서 코드 리뷰 한 번만 더 달아주실 수 있을까요?

---

[#aec899... ([Style] 코드리뷰 반영)](https://github.com/Team-EverFlow/Koreasy/pull/43/commits/aec89977ac2aba29e732b4706e0d6eae15cd38b8) 커밋에서 [src/pages/Wordtest/WordtestTitle.jsx](https://github.com/Team-EverFlow/Koreasy/pull/43/commits/aec89977ac2aba29e732b4706e0d6eae15cd38b8#diff-d700b20513aea2ffd61228f096d5250476da18ee78a48717d78ef82a8dd21a79) 파일 내에 있는 9번째 줄과 10번째 줄에서 `p`태그를 사용했습니다. 지금은 big-title과 small-title을 `p` 태그가 아닌 `div` 태그로 변경된 사실을 확인하였습니다.

그러나 보통 `div` 태그는 요소를 그룹으로 묶을 때 사용합니다. 저 상황에서는 그룹으로 묶는 것이 아닌 하나의 스타일을 정의한 것이기 때문에 `span` 사용을 권장하고 싶습니다. ([관련자료](https://blog.hubspot.com/website/span-vs-div#html))